library/build.gradle

```diff
@@ -1,6 +1,9 @@
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
 apply plugin: 'signing'
+apply plugin: 'com.github.dcendents.android-maven'
+
+group='com.github.WhistleLabs'
```

Comment:
The docs say it has to be of the form ` com.github.User:Repo:Tag`, haven't tested using anything else, but would imagine it probably has to be `com.github` for it to work. Can consolidate the lib into bintray or something when you get back maybe, this was just the path of least resistance during development.